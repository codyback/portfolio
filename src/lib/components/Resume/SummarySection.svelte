<script lang="ts">
	export let description: string;
</script>

<section
	id="cvSummary"
	class="mt-10 sm:mt-20"
>
	<img
		src=""
		alt="Cody Backus"
		class="rounded-sm lg:rounded-full h-48 lg:h-48 sm:h-96 lg:w-48 w-full bg-gray-500"
	/>
	<div class="mt-4 text-xl font-bold">Cody Backus</div>
	<p class="mt-10 text-neutral-500 text-justify">
		{description}
	</p>
</section>
