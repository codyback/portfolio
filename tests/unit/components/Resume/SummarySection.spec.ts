import SummarySection from '@/lib/components/Resume/SummarySection.svelte';
import { render } from '@testing-library/svelte';
import { vi } from 'vitest';

describe('Header', () => {
	beforeAll(() => {
		vi.resetAllMocks();
	});

	it('renders correctly', () => {
		const { container } = render(SummarySection, {
			description:
				'Summary Description - Ex exercitation deserunt veniam ipsum est non aute occaecat.',
		});
		expect(container).toMatchSnapshot();
	});
});
