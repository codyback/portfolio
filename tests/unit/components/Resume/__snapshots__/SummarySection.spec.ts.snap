// Vitest Snapshot v1

exports[`Header > renders correctly 1`] = `
<body>
  <div>
    <section
      class="mt-10 sm:mt-20"
      id="cvSummary"
    >
      <img
        alt="Cody Backus"
        class="rounded-sm lg:rounded-full h-48 lg:h-48 sm:h-96 lg:w-48 w-full bg-gray-500"
        src=""
      />
       
      <div
        class="mt-4 text-xl font-bold"
      >
        Cody Backus
      </div>
       
      <p
        class="mt-10 text-neutral-500 dark:text-neutral-300 text-justify"
      >
        Summary Description - Ex exercitation deserunt veniam ipsum est non aute occaecat.
      </p>
    </section>
    <!--&lt;SummarySection&gt;-->
  </div>
</body>
`;
